index.js

```diff
@@ -81,6 +81,19 @@ class StoresQuery {
     return this;
   }
 
+  /**
+   * Returns only stores in the given country.
+   * Countries are specified using to letter codes,
```

Comment:
Specifically, they are specified using [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements). I think there is no harm and potential help in mentioning that, so I would recommend amending this doc string with either a mention of or link to the ISO.